packages/deployer/test/sample-project/contracts/MainProxy.sol

```diff
@@ -0,0 +1,37 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./storage/ProxyNamespace.sol";
+
+contract MainProxy is ProxyNamespace {
+    function _fallback(address implementation) internal virtual {
```

Comment:
MainProxy defines the proxy on it, not extending any Proxy

Reply:
A bit new on the "virtual" keyword here too. Question: do we need it here, since I doubt we will be inheriting MainProxy anywhere.